test/modules/lighthouse_service.spec.js

```diff
@@ -241,7 +241,7 @@ describe('lighthouse_service api', () => {
           count_fit_to_pick_samples: 2,
         },
       ]
-      const response = { data: { plates } }
+      response = { data: { plates } }
```

Comment:
Was there a problem with these being `const`?

Reply:
`response` was just being declared at the top of the file already

Reply:
Ah right.  Should it be reassigned here then?  Better solutions would be to either use a new variable name here or in the one higher up.  I know these are only tests, but reusing variables leads to unexpected bugs.